attributes/default.rb

```diff
@@ -11,3 +11,6 @@
   fasttrack
   #{contrib}
 )
+
+# NOTE: if set to false, repositories starting with CentOS-SCLo-* won't be deleted
```

Comment:
`if set to false, repositories starting with CentOS-SCLo-* will be deleted`

Reply:
woops good catch